cmdstanpy/utils.py

```diff
@@ -121,11 +122,12 @@ def validate_cmdstan_path(path: str) -> None:
     Throws exception if specified path is invalid.
     """
     if not os.path.isdir(path):
-        raise ValueError('no such CmdStan directory {}'.format(path))
+        raise ValueError(
+            'No CmdStan directory, ' 'path {} does not exist.'.format(path)
```

Comment:
Why are the strings in this file split like this?

Reply:
I blame 'black'.  or gremlins.  fixed.